.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "eslint:recommended",
+    "plugin:jest/recommended",
+    "plugin:jest-dom/recommended",
+    "plugin:prettier/recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "@typescript-eslint/no-empty-interface": "off",
+    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }]
```

Comment:
This allows us to ignore unused arguments by prefixing them with `_`. Example:

https://github.com/Developer-DAO/web3-ui/blob/e7c55be63d4595a3bc977e6a8cd8ab3127f33e7a/packages/components/src/components/AddressInput/AddressInput.tsx#L37